Makefile

```
@@ -125,15 +125,20 @@ pkg/subctl/operator/common/embeddedyamls/yamls.go: pkg/subctl/operator/common/em
 	go generate pkg/subctl/operator/common/embeddedyamls/generate.go
 
 # Operator CRDs
-deploy/crds/submariner.io_servicediscoveries.yaml: ./apis/submariner/v1alpha1/servicediscovery_types.go vendor/modules.txt
-	controller-gen $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=deploy/crds
+CONTROLLER_GEN := $(CURDIR)/bin/controller-gen
+$(CONTROLLER_GEN): vendor/modules.txt
+	mkdir -p bin
```

```suggestion
	mkdir -p $$(dirname $@)
```

I guess something like this should be more robust?

---

Good idea, I’ll raise you a `mkdir -p $(@D)` ;-).

---

You're a true `make` wiz :D